src/test/java/core/basesyntax/BalanceOperationTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.strategy.BalanceOperation;
+import core.basesyntax.strategy.FruitTransaction;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class BalanceOperationTest {
```

Comment:
Let's add a negative test case with quantity less than 0. Please, add it for all handlers

Reply:
Should I also add it to the OperationHandler interface? Thanx!